### PR TITLE
Add decision to use compiled matlab

### PR DESCRIPTION
**Pull request by mducle**
Recorded decision to use a compiled Matlab library for the Python interface to PACE.

### CODE REVIEW

**Review by cmarooney-stfc — APPROVED** [review]
This is fine

File: documentation/adr/0009-use-compiled-matlab-for-python.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+Matlab has [built-in functionality](https://www.mathworks.com/help/matlab/call-python-libraries.html)
+to call Python programs however the reverse is not true - principally because
+Matlab requires a (paid-for) license to use.
+
+Thus to allow Python users to call the current Matlab PACE codes
+([Horace](https://github.com/pace-neutrons/Horace/)/[Herbert](https://github.com/pace-neutrons/Herbert/))
+it should either be translated to Python or C++ and a wrapper to Python and Matlab created,
+or the Matlab Compiler Runtime (MCR) toolbox should be used to "compile" 
+`horace`/`herbert` for distribution as a Python package, as detailed
+[here](../../python_interface/design/01_pace_python_high_level_discussion.md).
```
Comment (rebeccafair) [inline documentation/adr/0009-use-compiled-matlab-for-python.md]: It should mention somewhere that you can run compiled Matlab without a license. It's implied and is in the linked documents, but would probably help if it was mentioned explicitly.

File: documentation/adr/0008-pace-syntax-guiding-principles.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-[<-previous](0007-developer-scripts-storage-location.md) | next->
+[<-previous](0007-developer-scripts-storage-location.md) | [next->](0009-use-compiled-matlab-for-python.md)
```
Comment (rebeccafair) [inline documentation/adr/0008-pace-syntax-guiding-principles.md]: Don't forget to renumber this or the Brille ADR to 10 depending on which is merged first..

**Review by rebeccafair — APPROVED** [review]
Just some small things, otherwise looks good!

**Review by g5t — APPROVED** [review]

**Pull request merged**